SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1609,6 +1631,29 @@ void USpatialNetDriver::WipeWorld(const USpatialNetDriver::PostWorldWipeDelegate
 	}
 }
 
+const FRPCInfo* USpatialNetDriver::GetRPCInfo(UObject* Object, UFunction* Function) const
```

Comment:
I strongly dislike this function sitting in the NetDriver. I believe it should sit in the ClassInfoManager.

Reply:
Sounds good to me.

Reply:
Moved